test/new-e2e/tests/agent-platform/common/helper/unix.go

```diff
@@ -20,3 +20,30 @@ func (u *Unix) GetConfigFolder() string { return "/etc/datadog-agent/" }
 
 // GetBinaryPath return the datadog-agent binary path
 func (u *Unix) GetBinaryPath() string { return "/usr/bin/datadog-agent" }
+
+// GetConfigFileName return the config file name
+func (u *Unix) GetConfigFileName() string { return "datadog.yaml" }
+
+// GetServiceName return the service name
+func (u *Unix) GetServiceName() string { return "datadog-agent" }
+
+// UnixDogstatsd implement helper function for Dogstatsd on Unix distributions
+type UnixDogstatsd struct{}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
To statically check you implement the interface:
```suggestion
type UnixDogstatsd struct{}

var _ Helper = (*UnixDogstatsd)(nil)
```